fdbserver/DataDistribution.actor.cpp

```diff
@@ -641,36 +645,122 @@ struct DataDistributor : NonCopyable, ReferenceCounted<DataDistributor> {
 	}
 };
 
-ACTOR Future<Void> resumeAuditStorage(Reference<DataDistributor> self, AuditStorageState auditStates);
+void addAuditToAuditMap(Reference<DataDistributor> self, std::shared_ptr<DDAudit> audit) {
+	AuditType auditType = audit->coreState.getType();
+	UID auditID = audit->coreState.id;
+	TraceEvent(SevDebug, "AuditMapOps", self->ddId)
+	    .detail("Ops", "addAuditToAuditMap")
+	    .detail("AuditType", auditType)
+	    .detail("AuditID", auditID);
+	ASSERT(!self->audits[auditType].contains(auditID));
+	self->audits[auditType][auditID] = audit;
+	return;
+}
+
+std::shared_ptr<DDAudit> getAuditFromAuditMap(Reference<DataDistributor> self,
+                                              AuditType auditType,
+                                              UID auditID,
+                                              int retryCount) {
```

Comment:
Do we need this `retryCount` as a parameter, it is used only for logging?

Reply:
retryCount is used as a epoch.